backend/dataall/core/groups/api/resolvers.py

```diff
@@ -71,3 +72,13 @@ def list_cognito_groups(context, source, filter: dict = None):
         if group not in invited_group_uris:
             res.append({"groupName": group})
     return res
+
+
+def get_groups_for_user(context, source, userid):
+    envname = os.getenv('envname', 'local')
+    if envname in ['dkrcompose']:
```

Comment:
```suggestion
    if envname in ['local', 'dkrcompose']:
```

Reply:
Addressed this in the next update. Thanks!